docs/make_work.jl

```diff
@@ -160,38 +35,17 @@ end
 # Remove the module prefix
 Base.print(io::IO, b::Base.Docs.Binding) = print(io, b.var)
 
-# Sanitize paths
 
-function sanitize(a::AbstractString, n::AbstractString)
-  b = splitpath(replace(a, Regex(".*$(n)") => n))
-  return joinpath(b[1], b[3:end]...)
-end
-
-function sanitize_dst(a::AbstractString, n::AbstractString)
-  b = splitpath(replace(a, Regex(".*$(n)") => n))
-  return joinpath("build", b[1], b[3:end]...)
-end
-
-bla = normpath(joinpath(dirname(pathof(Hecke)), "..", "docs", "src"))
-const hecke = sanitize(bla, "Hecke")
-
-bla = normpath(joinpath(dirname(pathof(Nemo)), "..", "docs", "src"))
-const nemo = sanitize(bla, "Nemo")
-
-bla = normpath(joinpath(dirname(pathof(AbstractAlgebra)), "..", "docs", "src"))
-const aa = sanitize(bla, "AbstractAlgebra")
+const hecke = "Hecke"
+const nemo = "Nemo"
+const aa = "AbstractAlgebra"
 
 bib = CitationBibliography(joinpath(@__DIR__, "oscar_references.bib"), sorting = :nyt)
```

Comment:
I guess `bib` should also be set in `doit`, so that changes to the bibliography get reflected?